scripts/update-coins.sh

```diff
@@ -32,9 +32,12 @@ $SRC/cointool.py dump -p -d connect -o $DIST/coins.json \
     -E eth \
     -E erc20 \
 
+# Exclude coins that are not supported by any device model
+yarn tsx "$(dirname "${BASH_SOURCE[0]}")"/filterCoins.ts
+
 yarn prettier --write $DIST/coins.json
 
-$SRC/cointool.py dump -p -d connect -o $DIST/coins-eth.json \
+$SRC/cointool.py dump -p -o $DIST/coins-eth.json \
```

Comment:
btw don't eth coins need filtering too?

Reply:
There is nothing it would filter out (yet). I could call it twice, not sure what is better.

Reply:
right, ok, I don't know. hope there will never be such scenario where one evm is supported but the other one not